Mayhemfile

```diff
@@ -0,0 +1,6 @@
+project: rnshah9/jpeg-compressor
+target: jpge
```

Comment:
Looks like a typo?

Reply:
I might be missing something obvious. Which typo?

Reply:
jpge -> jpeg. I'll resolve.

Reply:
Ah wait, that's actually what the target is called.